commit 8880bdb67c9d939ec53506d05f5ce1d75be10c97
Author: nicm <nicm>
Date:   Wed Apr 2 17:08:23 2014 +0000

    Do not replace ## with # in status_replace1 because it'll be done later
    by the format code.

diff --git a/status.c b/status.c
index bf528b0..029be4c 100644
--- a/status.c
+++ b/status.c
@@ -396,9 +396,6 @@ status_replace1(struct client *c, char **iptr, char **optr, char *out,
 	case '{':
 		ptr = (char *) "#{";
 		goto do_replace;
-	case '#':
-		*(*optr)++ = '#';
-		break;
 	default:
 		xsnprintf(tmp, sizeof tmp, "#%c", *(*iptr - 1));
 		ptr = tmp;
